Project.toml

```diff
@@ -57,7 +57,7 @@ Requires = "1.1"
 SpecialFunctions = "1.6, 2"
 Statistics = "1"
 Tracker = "0.2"
-ZygoteRules = "0.2.5"
+ZygoteRules = "=0.2.5"
```

Comment:
This bound needs to be updated after https://github.com/FluxML/ZygoteRules.jl/pull/31.